source/src/notebook/healthcare-and-life-sciences/molecular-unfolding/molecular_unfolding.ipynb

```diff
@@ -0,0 +1,891 @@
+{
```

Comment:
<p>what about those commented lines?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/awslabs/quantum-computing-exploration-for-drug-discovery-on-aws/pull/180/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
<p>Let me put all these codes in one code block for clarity</p>

Reply:
<p>already make it more clear in commit</p>